overlays/ghc-packages.nix

```diff
@@ -67,8 +70,11 @@ in rec {
       (pkgName: dir: importCabal "${name}-${pkgName}" "${value.passthru.configured-src}/${dir}") ghc-extra-pkgs)
     self.buildPackages.haskell-nix.compiler;
 
-  ghc-extra-pkgs-cabal-projects = builtins.mapAttrs (name: value: let package-locs = builtins.mapAttrs (_: dir: "${value.passthru.configured-src}/${dir}") ghc-extra-pkgs; in
-    self.writeTextFile {
+  ghc-extra-pkgs-cabal-projects = builtins.mapAttrs (name: value:
+    let package-locs =
+      builtins.mapAttrs (_: dir: "${value.passthru.configured-src}/${dir}")
+        (self.lib.filterAttrs (n: _: n != "base" && n != "bytestring" && n != "ghc-heap") ghc-extra-pkgs);
```

Comment:
This will need a comment I believe.

Reply:
I'm hoping we can revert some of that now that we have a `ghc-8.8.1-reinstallable-lib-ghc.patch` (not sure about `base`, but maybe `bytestring` and `ghc-heap`).